nilearn/datasets/struct.py

```diff
@@ -51,7 +59,7 @@ def fetch_icbm152_2009(data_dir=None, url=None, resume=True, verbose=1):
         "t1", "t2", "t2_relax", "pd": anatomical images obtained with the
         given modality (resp. T1, T2, T2 relaxometry and proton
         density weighted). Values are file paths.
-        "gm", "wm", "csf": segmented images, giving resp. gray matter,
```

Comment:
Yes, I understand. Just for future reference in this PR, in this docstr and I only made pep8 changes because my editor was complaining and one orthographic change.